chart/templates/server-proxy-ssl-dhparam.yaml

```diff
@@ -1,3 +1,6 @@
+# Copyright (c) 2021 Gitpod GmbH. All rights reserved.
```

Comment:
@akosyakov, please revert this change, or fix the build otherwise.

@corneliusludmann, thank you for figuring this out! 🙏🏻